charts/istio-operator/templates/deployment.yaml

```
@@ -17,6 +17,9 @@ spec:
     {{- end }}
       labels:
         {{- include "istio-operator.selectorLabels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- tpl (toYaml .Values.podLabels) $ | nindent 8 }}
+        {{- end }}
```

Could you follow the chart idioms as suggested below, a discussion about templating these would be separate to adding the labels.

```suggestion
        {{- with .Values.podLabels }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```